addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1347,6 +1351,28 @@ var Wysiwyg = Widget.extend({
         }
         this.$attribute_translations = $('.o_editable_translatable_attribute');
         this.translations = [];
+        // (id -> value) map of the initial values of a translation, to be able to determine
+        // whether a translation has been modified or not, so as to not save unmodified translation
+        // (important because when a translation is saved, Odoo marks it as translated)
```

Comment:
You had no way to know that but we try to limit the comment lines to 80 chars maximum.